electron/game_config.ts

```diff
@@ -178,7 +178,7 @@ class GameConfigV0 extends GameConfig {
   }
 
   public async getSettings(): Promise<GameSettings> {
-    const settings = JSON.parse(readFileSync(this.path, 'utf-8'))
+    const settings = existsSync(this.path) ? JSON.parse(readFileSync(this.path, 'utf-8')) : {}
```

Comment:
So if there is no game settings file we just return an empty object? 
Before, if there is no game settings file we returned the default settings. Otherwise, this might break the launch of games and the loading of the settings.

Reply:
Yeah this needs fixing, my last commit is really quite broken.